packages/ingest/util.js

```diff
@@ -2,21 +2,6 @@
 
 const mime = require('mime-types');
 
-/**
- * Ensure provider path conforms to expectations.
- * Removes any/all leading forward slashes.
- *
- * @param {string} provPath - provider path
- * @returns {string} path, updated to conform if necessary.
- */
-function normalizeProviderPath(provPath) {
```

Comment:
This function was a bad idea way back when. Turns all relative paths absolute. Boooo. Who committed that?? ... 😰

Reply:
According to the "rules", we would need to deprecate this instead of just deleting it

Reply:
I don't know who else would use a function that is specifically written for messing with provider paths, but I would rather break them than have them continue to use a function that "normalizes" by making all the paths absolute, which is simply a terrible idea.

Reply:
But I can deprecate it and put some massive warning comments in.

Reply:
4a1835ded